core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.ts

```diff
@@ -17,11 +17,11 @@
  */
 
 import { Component, OnInit } from '@angular/core';
-import { SafeUrl } from '@angular/platform-browser';
 import { downgradeComponent } from '@angular/upgrade/static';
 import { UrlInterpolationService } from 'domain/utilities/url-interpolation.service';
 import { LanguageUtilService } from 'domain/utilities/language-util.service';
 import AppConstants from 'assets/constants';
+import { AppConstants as appConstants } from 'app.constants';
```

Comment:
Please remove the previous import and only do 
```suggestion
import { AppConstants } from 'app.constants';
```
`assets/constant` is a subset of `app.constants`

Reply:
Done